### PR TITLE
chore: Improve downloads script

### PR DESCRIPTION
**Pull request by florian-richter**
The current downloads action fails. I hopefully fixed it and improved the output a bit:

```
@sap-cloud-sdk/util:
- v1: 139809 downloads
- v2: 4687 downloads

@sap-cloud-sdk/connectivity:
- v2: 4019 downloads

@sap-cloud-sdk/http-client:
- v2: 3874 downloads

@sap-cloud-sdk/odata-v2:
- v2: 656 downloads

@sap-cloud-sdk/odata-v4:
- v2: 329 downloads

@sap-cloud-sdk/generator:
- v1: 465 downloads
- v2: 262 downloads

@sap-cloud-sdk/openapi:
- v2: 269 downloads

@sap-cloud-sdk/openapi-generator:
- v1: 80 downloads
- v2: 226 downloads

@sap-cloud-sdk/test-util:
- v1: 521 downloads
- v2: 278 downloads

@sap-cloud-sdk/eslint-config:
- v1: 109 downloads
- v2: 65 downloads

@sap-cloud-sdk/temporal-de-serializers:
- v2: 15 downloads

@sap-cloud-sdk/odata-common:
- v2: 711 downloads

@sap/cds:
- v1: 106 downloads
- v2: 92 downloads
- v3: 2393 downloads
- v4: 13691 downloads
- v5: 145101 downloads
```

### CODE REVIEW

File: scripts/npm-stats.ts [hunk 1]
```diff
@@ -39,21 +45,28 @@ async function getAllStats(page) {
...
     '@sap/cds'
-  ].reduce(
-    async (stats, packageName) => ({
-      ...(await stats),
-      [packageName]: await getStatsForPackage(page, packageName)
-    }),
-    {}
-  );
+  ].map(packageName => {
+    return getStatsForPackage(browser, packageName);
```
Comment (tomfrenken) [inline scripts/npm-stats.ts:49]: Why did you remove the `await` here? Other than that LGTM

**Review by tomfrenken — APPROVED** [review]
Just one small comment.

File: scripts/npm-stats.ts (hunk 1 above)
Comment (florian-richter) [reply scripts/npm-stats.ts:49]: The function returns an array of promises so that they can run in parallel. Reduce would run in sequence and await each response before continuing to the next element.
